app/src/ui/changes/commit-message.tsx

```diff
@@ -1148,9 +1148,11 @@ export class CommitMessage extends React.Component<
       return verb
     }
 
+    const action = `${verb} ${this.getFilesToBeCommittedButtonText()}to `
```

Comment:
Can we add a comment here or else I feel like I could just as easily come here and revert this change in the future thinking "why on earth is this not just in the jsx"

Reply:
Done!